### DISCUSSION TITLE
New room card state templates

### DISCUSSION THREAD

**Discussion by kepath** [UI-Lovelace-Minimalist/UI#1035] [Show and tell]
Hi,
I have created some state templates for use with, but not limited to, the room card.
These are similar to the <colour>_on and <colour>_off templates, but they set different colours and icons for different states.

A simple example would be for a printer, where the state 'ready' would be blue; anything else would be red
<img width="42" alt="Screenshot 2022-09-30 at 22 28 23" src="https://user-images.githubusercontent.com/35769141/193359072-5d799584-3d46-4b5c-982c-113988bbbbfb.png">
I have several others - washing machine activity,
<img width="48" alt="Screenshot 2022-09-30 at 22 35 26" src="https://user-images.githubusercontent.com/35769141/193359891-a9f597c7-d798-4051-8e3d-3ba235f9679e.png">
weather predictions with the colour representing the temperature,
<img width="193" alt="Screenshot 2022-09-30 at 22 35 12" src="https://user-images.githubusercontent.com/35769141/193359915-647f9054-aee9-4658-9ab8-9b9101f273aa.png">
robot mower status with a rain icon if rain is detected,
<img width="48" alt="Screenshot 2022-09-30 at 23 06 43" src="https://user-images.githubusercontent.com/35769141/193362915-b4120544-bd23-421a-839d-f47d2bf127a6.png">
etc

However I do not think these state templates fit into the existing directory structure. 
Should I create a 'states' folder at the same level as 'colours'?
Or perhaps create a 'card_room_templates' directory in the same location as card_room.yaml?

Does an admin have any thoughts/guidance on this?

**Comment by antonio1475** [comment]
Can you share some examples?
I have something similar: a green_playing for status:playing and they just sit in `/config/ui_lovelace_minimalist/custom_cards`


![image](https://user-images.githubusercontent.com/63130724/203799706-e4f77b6c-e691-46bf-ba6b-efda4645bb81.png)
